react-cors-spa/src/APIDisplay.js

```diff
@@ -0,0 +1,60 @@
+import './ApiDisplay.css';
+import { useQuery } from "@tanstack/react-query";
+import axios from 'axios';
+
+// Run axios get request for quote
+const fetchQuotes = async () => {
```

Comment:
As a general rule of thumb, these API call methods usually go in another file so they can be reused, but since this is temporary just leave it here.